src/components/about.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+	import type { License } from '$lib/license';
+	import { t } from '$lib/stores/i18n';
+
+	async function loadLicenses() {
+		const response = await fetch('/licenses.json');
+		return (await response.json()) as License[];
```

Comment:
since .json() returns any you don't need to cast and can just clarify the functions return type to be Promise<License[]>